### ISSUE TITLE
Issue in page Deploying Scylla Monitoring Stack Without Docker

### ISSUE THREAD

**Issue by GarrettPoore** [scylladb/scylla-monitoring#1433]
I would like to report an issue in page https://monitoring.docs.scylladb.com/branch-3.7/install/monitor_without_docker

### Problem
In the following Note:
> Confirm before beginning that your Grafana and Prometheus versions are supported by the Monitoring Stack version you want to install. See the [Scylla Monitoring Stack Compatibility Matrix](https://monitoring.docs.scylladb.com/install/monitoring_stack/#prerequisites).

The Compatibility Matrix is a dead link.

Current link: https://monitoring.docs.scylladb.com/install/monitoring_stack/#prerequisites

###  Suggest a fix
Correct link: https://monitoring.docs.scylladb.com/stable/install/monitoring_stack.html#prerequisites

**Comment by tzach** [comment]
The link is defined as relative 
```
 `Scylla Monitoring Stack Compatibility Matrix </install/monitoring_stack/#prerequisites>`_.
```
and works fine locally in `make preview`, but break in multi version

**Comment by tzach** [comment]
@dgarcia360 can you please take a look?
I suspect its related to the local `#prerequisites` part

**Comment by amnonh** [comment]
Fixed by #1438
